src/utils/__tests__/index.test.js

```diff
@@ -85,3 +85,39 @@ describe('checkTemplateConfigFile', () => {
     }).not.toThrow();
   });
 });
+
+describe('getTemplatesByCategory', () => {
+  beforeAll(() => {
+    mockReaddirSync.mockImplementation(() => [
+      'InstantSearch.js',
+      'Angular InstantSearch',
+      'React InstantSearch',
+      'Vue InstantSearch',
+      'React InstantSearch Native',
+      'InstantSearch iOS',
+      'InstantSearch Android',
+    ]);
+    mockLstatSync.mockImplementation(() => ({ isDirectory: () => true }));
+  });
+
+  test('return the correct templates and categories', () => {
+    expect(utils.getTemplatesByCategory()).toEqual({
+      Web: expect.arrayContaining([
```

Comment:
Why do we use `arrayContaining` on this assertion? We check for an exact match not a partial one.

Reply:
I use `arrayContaining` because we don't care about the order. Does that make sense?

Reply:
Yep, just for curiosity the order is not stable?

Reply:
I think it depends on your OS because it's determined by `fs.readdirSync()` (or perhaps Node standardize that). It should be the same everywhere but I suppose there's no guarantee. We could add some weights if it's problematic.